### ISSUE TITLE
Shortcuts for MAC.

### ISSUE THREAD

**Issue by reidark** [jrowny/brackets-snippets#53]
Hi guys.

I've read about the shortcuts for MAC. Some people are using the Shift-Tab, but this shortcut is for ident/reident. I tried other shortcuts, but not working at all, even the Shift-Tab.

How i can fix this?

**Comment by zaggino** [comment]
You can set-up your own shortcuts in the settings dialog (open from snippets panel).

**Comment by reidark** [comment]
@zaggino Yep, but all of the shortcuts i've tried to set did not worked, for some reason that i don't know.

**Comment by jsbalrog** [comment]
I am having the same problem here, even after redefining my shortcut in the settings from the snippets panel.

**Comment by vinyll** [comment]
Same here on 2 Mac's

**Comment by he-jlalonde** [comment]
Same here... is it fixed for some of you? How did you fixed it?

**Comment by zaggino** [comment]
@he-jlalonde I've started my own rewrite from scratch here https://github.com/zaggino/brackets-snippets - you can find it in extension registry under `Brackets Snippets (by zaggino)`. Development of this extension has been abandoned for months now and it doesn't work in latest Brackets as far as I know.

**Comment by he-jlalonde** [comment]
Ok thank you zaggino!
